addons/popochiu/engine/objects/inventory_item/popochiu_inventory_item.gd

```diff
@@ -352,6 +354,8 @@ func _on_gui_input(event: InputEvent) -> void:
 		MOUSE_BUTTON_LEFT:
 			if I.active:
 				await on_item_used(I.active)
+				# after item has been used return to normal state
+				I.active = null
```

Comment:
Is this needed? You're already changing the current active inventory item inside the `on_item_used` function.